src/main/java/org/app/dto/RegisterUserResponseDto.java

```diff
@@ -8,6 +8,6 @@
 @Builder
 public class RegisterUserResponseDto {
     private String username;
-    private String password;
+    private String password; // todo
```

Comment:
When registering, I received this response
```
{
  "username": "john3@example.com",
  "password": "$2a$10$hS.SmmsgMIZ6XNlUCpgefOm1zJTK8f85V7imi2C45snSZpBCelnnO",
  "role": "CLIENT"
}
```

Exposing password hashes to the user is a security violation